tests/tnt/concurrent_topology_test.go

```diff
@@ -115,7 +116,13 @@ func TestConncurrentTopologyChange(t *testing.T) {
 		return
 	}
 
-	t.Parallel()
+	// suppress the below linter warning:
+	// unused-parameter: parameter 't' seems to be unused, consider removing or renaming it as _ (revive)
+	_ = t
```

Comment:
`t` is passed to the `require.Nil()` method, this line is no longer needed

Reply:
Done